### ISSUE TITLE
Create baseline model - ValueError: too many values to unpack (expected 2)

### ISSUE THREAD

**Issue by claudio-toledo** [jdb78/pytorch-forecasting#230]
- PyTorch-Forecasting version: 0.7.1
- PyTorch version: 1.7.1
- Python version: 3.7
- Operating System: MAC OS Big Sur: Version 11.1

### Expected behavior

I executed code 

actuals = torch.cat([y for x, (y, weight) in iter(val_dataloader)])
baseline_predictions = Baseline().predict(val_dataloader)
(actuals - baseline_predictions).abs().mean().item()

in order to follow along the documentation/tutorial in the Create baseline model and expected to get result 293.0088195800781

### Actual behavior

However, result was 

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-9-894a331e72a6> in <module>
----> 1 actuals = torch.cat([y for x, (y, weight) in iter(val_dataloader)])
      2 baseline_predictions = Baseline().predict(val_dataloader)
      3 (actuals - baseline_predictions).abs().mean().item()

<ipython-input-9-894a331e72a6> in <listcomp>(.0)
----> 1 actuals = torch.cat([y for x, (y, weight) in iter(val_dataloader)])
      2 baseline_predictions = Baseline().predict(val_dataloader)
      3 (actuals - baseline_predictions).abs().mean().item()

ValueError: too many values to unpack (expected 2)

I think it has to do with 

torch.cat([y for x, (y, weight) in iter(val_dataloader)])

because of the value of broadcasting the iter of the variable. It should be 

torch.cat([y for (x, y) in iter(val_dataloader)])

### Code to reproduce the problem

```
actuals = torch.cat([y for x, (y, weight) in iter(val_dataloader)])
baseline_predictions = Baseline().predict(val_dataloader)
(actuals - baseline_predictions).abs().mean().item()

```

Paste the command(s) you ran and the output. Including a link to a colab notebook will speed up issue resolution.
If there was a crash, please include the traceback here.
The code used to initialize the TimeSeriesDataSet and model should be also included.

**Comment by jdb78** [comment]
I think that is because you are using the current master which contains a breaking change. I have not updated the tutorial yet.

**Comment by jdb78** [comment]
Should be fixed with the latest version of the package 0.8.0. Tutorials are in sync.

**Comment by kaiharuto** [comment]
Running `pytorch-forecasting==0.8.2` this error also occurs for me
